templates/html-header.twig

```diff
@@ -7,6 +7,13 @@
 	{% include 'blocks/meta_fields.twig' %}
 
 	{% if google_tag_value %}
+		{% if ab_hide_selector %}
+		<style>
+			.google-optimize-loading {{ ab_hide_selector }} {
```

Comment:
This does allow entering things that mess up the HTML, but that is also the case for other existing fields (like google_tag_value).